ntrans_gui.py

```diff
@@ -262,21 +298,25 @@ def update_estimated_time_label(self, *junk: object) -> None:
         total_time_in_seconds = int(
             self.data_size_var.get()
             * len([n for n, var in self.checkbox_vars.items() if var.get()])
-            * 1.2  # Average time, sec per string
-        )
-        if total_time_in_seconds >= 60:
+            * 1.2
+        )  # Average time in seconds per N-gram
+        if total_time_in_seconds >= 60 * 60:
```

Comment:
If you want, you can let `datetime.timedelta` do the math for you:

```
>>> import datetime
>>> str(datetime.timedelta(seconds=5))
'0:00:05'
>>> str(datetime.timedelta(seconds=12345))
'3:25:45'
```

The format is `hours:minutes:seconds`, and you can `.split(":")` if you want to separate them.

Reply:
Will start working on this now!